sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestUseClasspathSqlLocator.java

```diff
@@ -71,4 +73,19 @@ public void testCachedOverride() {
     @RegisterRowMapper(SomethingMapper.class)
     @UseClasspathSqlLocator
     public interface SubCromulence extends Cromulence {}
+
+    @Test
+    public void emptyFileThrows() {
+        assertThatThrownBy(handle.attach(Blanks.class)::methodWithEmptyFile)
+            .isInstanceOf(UnableToCreateStatementException.class)
+            .hasMessageContaining("the query returned")
+            .hasMessageContaining("methodWithEmptyFile()")
+            .hasMessageContaining("was empty");
+    }
+
+    @UseClasspathSqlLocator
+    public interface Blanks {
+        @SqlQuery("empty")
```

Comment:
I thought this was testing an error if the file is not located.
This seems to test that the file is located but empty -- it's not as clear to me that this is an error; the user has explicitly configured a valid but empty template.